src/main/java/com/onarandombox/MultiverseCore/commands/DumpsCommand.java

```diff
@@ -0,0 +1,242 @@
+package com.onarandombox.MultiverseCore.commands;
+
+import co.aikar.commands.CommandIssuer;
+import co.aikar.commands.annotation.*;
+import com.dumptruckman.minecraft.util.Logging;
+import com.onarandombox.MultiverseCore.MultiverseCore;
+import com.onarandombox.MultiverseCore.api.MVWorldManager;
+import com.onarandombox.MultiverseCore.commandtools.MVCommandManager;
+import com.onarandombox.MultiverseCore.commandtools.MultiverseCommand;
+import com.onarandombox.MultiverseCore.commandtools.flags.CommandFlag;
+import com.onarandombox.MultiverseCore.commandtools.flags.CommandFlagGroup;
+import com.onarandombox.MultiverseCore.commandtools.flags.ParsedCommandFlags;
+import com.onarandombox.MultiverseCore.config.MVCoreConfig;
+import com.onarandombox.MultiverseCore.event.MVVersionEvent;
+import com.onarandombox.MultiverseCore.utils.webpaste.PasteFailedException;
+import com.onarandombox.MultiverseCore.utils.webpaste.PasteService;
+import com.onarandombox.MultiverseCore.utils.webpaste.PasteServiceFactory;
+import com.onarandombox.MultiverseCore.utils.webpaste.PasteServiceType;
+import jakarta.inject.Inject;
+import org.apache.commons.lang.StringUtils;
+import org.bukkit.entity.Player;
+import org.bukkit.scheduler.BukkitRunnable;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jvnet.hk2.annotations.Service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@CommandAlias("mv")
+public class DumpsCommand extends MultiverseCommand {
+
+    private final MVCoreConfig config;
+    private final MultiverseCore plugin;
+    private final MVWorldManager worldManager;
+
+    @Inject
+    public DumpsCommand(@NotNull MVCommandManager commandManager,
+                        @NotNull MultiverseCore plugin,
+                        @NotNull MVCoreConfig config,
+                        @NotNull MVWorldManager worldManager) {
+        super(commandManager);
+        this.config = config;
+        this.plugin = plugin;
+        this.worldManager = worldManager;
+
+        registerFlagGroup(CommandFlagGroup.builder("mvdumps")
+                .add(CommandFlag.builder("--pastebincom")
```

Comment:
I think --pastebin good enough, don't need com

Reply:
I did think that originally but I don’t people using —pagebin to put it on a pastebon, not realising it actually means pastebin.com

Reply:
huh then what it means?

Reply:
Upload to https://pastebin.com